qa_bands.md

```diff
@@ -50,54 +57,51 @@
-* You are an active advocate for clarity in technical approach.
+* You are setting/establishing best practices/patterns at a QA software level (testing practices, risk identification and management, coding practices, style guidelines, core libraries, etc.) in the interest of producing clean, maintainable automation code.
 * You are helping recognize opportunities for simplification.
 * You are seen as a mentor and effective partner in helping others identify mistakes, pitfalls, risks, and alternative solutions.
 * You understand how your product is functioning/performing in production and bring a metrics focus to your job.
-* You are identifying new opportunities to streamline/optimize/improve our workflow, codebase, quality techniques, tools, and processes.
+* You are identifying new opportunities to streamline/optimize/improve our workflow, automation codebase, quality techniques, tools, and processes.
 * You have strong relationships within and outside of the Engineering team.
-* You have a sense of who key stakeholders are and who needs to be informed/when they need to be informed.
 
 <a name="staff-qa-engineer"></a>
 ## Staff QA Engineer
 ### Technical Abilities
-* Mastery of the testing techniques in the project they work in
+* Mastery of the testing techniques (manual and automation)in the project they work in
 * Possesses refined point of view on different testing approaches, optimizations and automation techniques
-* Rapid learner that quickly ramps on new languages, frameworks, and systems
-* Strong understanding of TuneIn's application and systems architecture
-* Identifies opportunities and approaches for improving Engineering productivity
+* Rapid learner that quickly ramps on new languages and testing frameworks
+* Strong understanding of TuneIn's application across multiple platforms
+* Identifies opportunities and approaches for improving QA productivity
 * Experienced with a wide variety of testing approaches, patterns, and implementation styles
 
 ### Project Scope
 * Own testing effort and process for a few large components or projects
-* Designs solutions to high complexity problems, with high-availability and high-reliability in mind
+* Design test automation for  high complexity problems, with high-availability and high-reliability in mind
 * Instigates test design discussions, evaluates approaches, articulates goals and criteria
-* Responsible for overall quality of code, product
+* Responsible for overall quality of code for automation and the application
 * Determines appropriate tools & techniques for monitoring quality of code in production and diagnosing issues
 * Primary driver of investigating and coordinating resolution for production issues
 * Ensures proper diagnosis and approach for resolving tricky bugs
```

Comment:
Perhaps would add one about co-driving investigation and resolution of nasty and/or high-priority bugs?